gpiozero/mixins.py

```diff
@@ -348,13 +342,10 @@ def __init__(self, *args, **kwargs):
         self._hold_thread = HoldThread(self)
 
     def close(self):
-        if getattr(self, '_hold_thread', None):
+        if self._hold_thread is not None:
```

Comment:
Hmmm, so is #507 no longer an issue then? :man_shrugging:

Reply:
What do you mean? #507 is a merged PR - do you mean the comment about `hasattr`?

Reply:
I meant that #507 changed this from `if self._hold_thread:` to `if getattr(self, '_hold_thread', None):`, and this PR now changes it back to `if self._hold_thread is not None:`
(i.e. it might explode if `close()` gets called on an object which doesn't have a `_hold_thread` attribute yet)